src/test/java/unstable/ContextMenuUnstableTests.java

```diff
@@ -0,0 +1,103 @@
+package unstable;
+
+import guitests.UITest;
+import javafx.application.Platform;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.input.KeyCode;
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.GuiTest;
+import ui.components.FilterTextField;
+import ui.components.IssueListView;
+import ui.listpanel.ListPanel;
+import ui.listpanel.ListPanelCell;
+import java.lang.reflect.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class ContextMenuUnstableTests extends UITest {
+
+    private static final String PANEL_IDENTIFIER = "#dummy/dummy_col0";
+
+    @Before
+    public void setup() {
+        Platform.runLater(stage::show);
+        Platform.runLater(stage::requestFocus);
```

Comment:
You probably want `runAndWait` here, and just one call, with both method calls inside.